infrastructure/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "product" {
+  type = "string"
+  default = "pdf-service"
+}
+
+variable "location" {
+  type = "string"
+  default = "UK South"
+}
+
+variable "env" {
+  type = "string"
+}
+
+variable "s2s_url" {
+  type = "string"
```

Comment:
Terraform will halt here because the variable isn't passed in by the pipeline and there is no default value. The pipeline only passes in the env and ilbIp - all other variables need to have a (default) value